tools/wptrunner/wptrunner/executors/executorwebdriver.py

```diff
@@ -623,6 +623,16 @@ def do_testharness(self, protocol, url, timeout):
                 if not is_alive:
                     raise Exception("Browser crashed during script execution.")
 
+            # A user prompt created after starting execution of the resume
+            # script will resolve the script with `null` [1, 2]. In that case,
+            # cycle this event loop and handle the prompt the next time the
+            # resume script executes.
+            #
+            # [1]: Step 5.3 of https://www.w3.org/TR/webdriver/#execute-async-script
+            # [2]: https://www.w3.org/TR/webdriver/#dfn-execute-a-function-body
+            if result is None:
+                continue
+
             done, rv = handler(result)
```

Comment:
Looks like the previous code is anyway not correct when `result` is None. I am not sure if we can make a change to `handler` to return 'False, None` if result is None.

There are other Executor classes calling the resume script, do we need to make a fix there also?